src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2308,6 +2312,163 @@ In case of PAM users, the login file should not be modified as password authenti
+
+===== Configuring ProActive properties for Keycloak
+The configuration file config/authentication/keycloak.cfg must be edited in order the set the proper values of the following properties:
+
+* The property `keycloak.url` should be set to the URL of the Keycloak server (already configured for ProActive).
+* The property `keycloak.https.allow_any_certificate` must be set to true when the keycloak server operates in TLS mode and uses a non-trusted (e.g., self-signed) TLS/SSL certificate. Otherwise, this property is set to false.
+* The property `keycloak.realm` must take the name of the Keycloak realm in which the ProActive client application is configured.
+* The property `keycloak.proactive.client_id` must take as value the name of the ProActive client application in Keycloak.
+* The property `keycloak.proactive.client_secret` must take as value the secret credential of the ProActive client application in Keycloak.
+* The property `keycloak.proactive.scope` must take as value the name of a scope configured in Keycloak. A scope defines the set of user information/parameters to be included in a Keycloak access token. If the value is empty, the default scope in Keycloak is used.
+* The property `keycloak.public.key` must have as value the content of public encryption key of Keycloak (e.g., public RSA key). This key is used to decrypt the access token returned by Keycloak.
+* The property `keycloak.principal_attribute` is the parameter in Keycloak access token based on which the user name is determined (e.g., `name` or `given_name`).
+* The property `keycloak.use_resource_role_mappings` is set to true when the user roles are determined based on the information block `resource_access` in Keycloak access token. When it is set to false, the user roles are determined based on the information block `realm_access` in Keycloak access token.
+* Using the fall back to file authentication: ProActive can be configured to use simultaneously file-based authentication and Keycloak-based access control. In this scenario, ProActive Scheduler checks at first user credentials (login/password) and group memberships in
+the login and group files, as performed in the <<_file,File login method>>. If the user or group attribution is not found in the login/group files, the user credentials and roles will be checked via Keycloak. There are three configuration options to control the fallback mechanism:
+** `pa.keycloak.authentication.fallback`: if set to `true`, the user credentials will be checked first in the login file, and then in the Keycloak server. If set to `false`, only the Keycloak server will be queried.
+** `pa.keycloak.group.membership.fallback`: if set to `true`, group memberships will be read from the Keycloak server if the user exists, and then from the group file. In that sense, a user will be affected to groups defined in both file and Keycloak server. If set to `false`, only the Keycloak server will be queried.
```

Comment:
should also be set to true by default

Reply:
I mentioned: 
If set to `true` (which is the default value)